pelican/tools/pelican_import.py

```diff
@@ -285,7 +285,7 @@ def dc2fields(file):
 
     print("%i posts read." % len(posts))
 
-    settings = read_settings()
+    settings = DEFAULT_CONFIG
     subs = settings['SLUG_REGEX_SUBSTITUTIONS']
```

Comment:
I kind of want to nitpick the unnecessity of the variable `settings`. This is the only time it is used:

```python
subs = DEFAULT_CONFIG['SLUG_REGEX_SUBSTITUTIONS']
```